src/hoc/advancedExpandTable/index.js

```diff
@@ -0,0 +1,199 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import set from 'lodash.set'
```

Comment:
Should these have been added to the package.json @mayank23 ?

Reply:
Likely. I get an error when using this HOC:
```
Module not found: Can't resolve 'lodash.get' in '/Users/cwack/code/branchweb/node_modules/react-table/lib/hoc/advancedExpandTable'
```

Reply:
Hey @cwackerfuss I don't think the maintainer is going to accept any PRs soon, so it's probably best to fork it and add it to the dependencies. Or you can just copy over the file and add it to your project. I'll see If I can get a standalone repo up soon too. Thanks 😄